### PR TITLE
add encryption info

### PR DESCRIPTION
**Pull request by nataliejeremy**
This closes issue #443

### CODE REVIEW

**Comment by bmackinney** [comment]
@jerryblakley Do you have anything to add about how this might affect sites?

File: source/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ category:
 ---
 SSL is a standard for establishing an encrypted link between your Pantheon site and a client (e.g. web browser). You should enable SSL on a custom domain, e.g., www.example.com, if you are transmitting any sensitive data. Loading a valid OpenSSL certificate into a Pantheon environment provisions an SSL load balancer with a dedicated IP address, allowing secure communication over HTTPS.
 
-**Note**: Enable SSL before updating DNS. SSL for custom domains is available for Professional plans and above.
+**Important**: Enable SSL before updating DNS. SSL for custom domains is available for Professional plans and above.
+
+**Note**: All traffic within the Pantheon infrastructure, from Varnish to application containers, is encrypted.
```
Comment (ari-gold) [inline source/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication.md]: How about moving the new note to the first paragraph? "...allowing secure communication over HTTPS. All traffic within the Pantheon Infrastructure, from Varnish to application containers, is encrypted."

I feel like having "Important" and then "Note" gives more noise and detracts from the fact that they need to enable SSL before updating DNS.

**Comment by nataliejeremy** [comment]
Thanks @ari-gold

**Comment by ari-gold** [comment]
+1

**Comment by jerryblakley** [comment]
+1 Note that non-new free sites do not yet have these encrypted links. That will happen when the servers get rotated over the course of I think a few months. Also, there may be a few straggler paid sites that didn't get moved to encrypted links.

**Pull request merged**